federationsender/queue/destinationqueue.go

```diff
@@ -186,7 +277,7 @@ func (oq *destinationQueue) nextInvites() bool {
 				"state_key":   ev.StateKey(),
 				"destination": oq.destination,
 			}).WithError(err).Error("failed to send invite")
-			continue
+			return false, err
```

Comment:
This seems like it could be a problem? It was `continue` before.

Reply:
I think this possibly highlights another issue, which is that if *any* of the invites fail, we'll retry all of them on the next run of the destination queue. I don't think this is causing the problem though.